engines/suppliers/app/models/suppliers/suppliers_finder.rb

```diff
@@ -0,0 +1,16 @@
+module Suppliers
+  class SuppliersFinder
```

Comment:
If we are thinking about using this `ByGroupFinder` all over the application and not only in `Suppliers` engine we should add it as a module in `Shared` engine and include it wherever we need it. Like we do with authentication and authorization modules.